components/ai_chat/resources/page/components/conversation_list/index.tsx

```diff
@@ -39,45 +35,58 @@ function ConversationList (props: ConversationListProps) {
   }, [props.list.length, props.isLoading, lastConversationEntryElementRef.current?.clientHeight])
 
   return (
-    <div className={styles.list}>
+    <div>
+      <div className={styles.list}>
       {props.list.map((turn, id) => {
-        const turnClass = classnames({
-          [styles.turnAI]: turn.characterType === CharacterType.ASSISTANT,
-          [styles.turnHuman]: turn.characterType === CharacterType.HUMAN,
-        })
-
         const isLastEntry = (id === (props.list.length - 1))
         const isLoading = isLastEntry && props.isLoading
         const elementRef = isLastEntry
           ? lastConversationEntryElementRef
           : null
 
+        const isHuman = turn.characterType === CharacterType.HUMAN
+        const isAIAssistant = turn.characterType === CharacterType.ASSISTANT
+
+        const turnClass = classnames({
+          [styles.turnAI]: isAIAssistant,
+          [styles.turnHuman]: isHuman,
+        })
+
+        const avatarStyles = classnames({
+          [styles.avatarAI]: isAIAssistant,
+          [styles.avatarHuman]: isHuman
+        })
+
         return (
-          <div key={id} ref={elementRef} className={turnClass}>
-            <p>
+          <div key={id} ref={elementRef} className={turnClass} data-turn={isHuman ? 'human' : 'assistant'}>
```

Comment:
Why do we need both the class name which is `turnAI|turnHuman` _and_ the data attribute which is `human|assistant`. It made it difficult to see in the css that they are the same element.

Reply:
makes sense. removed.